Find a functionally equivalent code:android.graphics.drawable.BitmapDrawable.getBitmap:COMMENT
Method Modifier: public      final       
Comment:/**
 * Returns the bitmap used by this drawable to render. May be null.
 */

@@ -1,3 +1,3 @@
 {
-    return mBitmap;
+    return mBitmapState.mBitmap;
 }

